code/modules/surgery/revival.dm

```diff
@@ -27,7 +27,10 @@
 
 /datum/surgery_step/revive
 	name = "shock brain"
-	implements = list(/obj/item/shockpaddles = 100, /obj/item/melee/baton = 75, /obj/item/gun/energy = 60)
+	implements = list(
+		/obj/item/shockpaddles = 100, //this is useful for reviving simepeople.
+		/obj/item/melee/baton = 20, //there's no reason to use a defib with this chance being high - so keep it low.
```

Comment:
blacklist stunprods

Reply:
I don't think I'm able to blacklist things in these lists, could just make only a specific type I suppose

Reply:
Actually what if I change it to like. an inducer.

Reply:
stunprods still need a high powered cell so its somewhat earned

Reply:
you can get these "high powered" cells from literally anything, including smeses which is a machine in almost every ship and ruin

I think inducers is better

Reply:
I cannot imagine the disappointment of someone who never used ghetto surgery when they use it as a last ditch effort to save their only other crewmember and they find out that they literally cannot use the stun prod which obviously shocks people to shock the brain. Using revival surgery should give you trauma or something so it comes at a cost instead of making it impossible to do in ghetto circumstances because it uses tools that can pretty much only be obtained using rnd at which point it stops being ghetto. I'm a medship main and i obviously would want PR's that make people more dependent on my ship but you simply cannot assume that there will always be a medship around to do the surgery properly

Reply:
You can always buy a surgery kit at the outpost for not that much really